packaging/linux/openra.appdata.xml

```diff
@@ -1,48 +1,60 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>openra.desktop</id>
-  <pkgname>openra</pkgname>
-  <name>OpenRA</name>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
`CC0-1.0` seems to be the default choice for these metadata files.  Are you okay with this choice @Mailaender?